src/PlaneWaveBasis.jl

```diff
@@ -138,6 +128,15 @@ function PlaneWaveBasis(model::Model{T}, Ecut::Number,
         kweights, ksymops, fft_size, grids, opFFT, ipFFT, opIFFT, ipIFFT
     )
 end
+function PlaneWaveBasis(model::Model, Ecut::Number, kgrid_size::AbstractVector;
```

Comment:
Do we want to support a number for the third argument to be the kpoint density? Or is that too tricky? Also for Ecut do we want to accept a 3-tuple of cutoffs for every direction, for easier support of 1d systems?

Reply:
For me that really falls into "advanced use", actually. For 1D you can also just explicitly specify the fft_size (and then you have even more control what happens). That's not disabled by the current interface. The kpoint density should be feasible.

Reply:
For 1D for sure that's advanced use. kpoint density is more user-friendly than kgrid_size, so maybe we support k density and leave the Ecut to the fft_size.

Reply:
but it's a bit tricky that having `[2,2,2]` as argument is different from `2` so maybe that's too magical

Reply:
Maybe I misunderstood your previous comment. I could leave it as is now. Is that fine?

Reply:
I like kwargs but that's a bit tricky. I would naively expect no info => no kpoint. Also we have to be careful that the user doesn't pass both kspacing and kgrid. That does look too complicated, so let's just revert to what it was before (sorry!), and the user can just call `PWBasis(model, Ecut, kgrid_size_from_minimal_spacing(lattice, spacing))`

Reply:
fair enough ... even though it would be nice to have a method, which automatically does sane defaults for kgrid and Ecut, actually ...

Reply:
Yeah; that we probably want to build on top. Ideally, the interface would be to just give an accuracy epsilon and the code would figure out the rest, but we're not there yet